scripts/examples-runner.ts

```diff
@@ -0,0 +1,23 @@
+import {extname, join} from "path";
```

Comment:
can you run prettier on this file according to our .prettierrc rules at root

so here would be no semicolons and spaces between the curly braces

thanks, looks good otherwise!

Reply:
Sure will do! Also will configure the formatted of my editor to use the monorepo config